### PR TITLE
SONARGRADL-128 Put all Gradle Build scripts in root sonar.sources only

### PR DESCRIPTION
**Pull request by leonardo-pilastri-sonarsource**

### CODE REVIEW

File: src/test/groovy/org/sonarqube/gradle/GradleKtsTests.groovy [hunk 1]
```diff
@@ -186,11 +186,11 @@ class GradleKtsTests extends Specification {
         props.load(outFile.newDataInputStream())
 
         then:
-        props[":subproject.sonar.sources"] == subProjectBuildFile.toRealPath().toString()
+        props[":subproject.sonar.sources"] == ""
 
         def sonarSources = props["sonar.sources"].split(",")
         Assertions.assertThat(sonarSources)
-                .containsExactlyInAnyOrder(settingsFile.toRealPath().toString(), buildFile.toRealPath().toString())
+            .containsExactlyInAnyOrder(settingsFile.toRealPath().toString(), buildFile.toRealPath().toString(), subProjectBuildFile.toRealPath().toString())
```
Comment (dorian-burihabwa-sonarsource) [inline src/test/groovy/org/sonarqube/gradle/GradleKtsTests.groovy:193]: How about placing one entry per line for readability?
```suggestion
            .containsExactlyInAnyOrder(
              settingsFile.toRealPath().toString(),
              buildFile.toRealPath().toString(),
              subProjectBuildFile.toRealPath().toString()
            )
```

File: src/test/projects/java-multi-module/userHome/caches/journal-1/file-access.properties
Comment (dorian-burihabwa-sonarsource) [inline src/test/projects/java-multi-module/userHome/caches/journal-1/file-access.properties:1]: Is this file supposed to be commited?

**Review by dorian-burihabwa-sonarsource — APPROVED** [review]
Thanks for the changes. Let's just confirm that `file-access.properties` needs to be present for our tests to work!

File: src/test/projects/java-multi-module/userHome/caches/journal-1/file-access.properties
Comment (leonardo-pilastri-sonarsource) [reply src/test/projects/java-multi-module/userHome/caches/journal-1/file-access.properties:1]: No indeed, thank you for catching that, it slipped through

**Review by leveretka — APPROVED** [review]
LGTM!

Just check the QG. You can dismiss or fix the issue.

**Pull request merged**